libs/ledgerjs/packages/hw-app-algorand/README.md

```diff
@@ -51,3 +51,18 @@ cosmos.getAddress("44'/283'/0'/0/0").then(o => o.address)
 ```
 
 Returns **[Promise](https://developer.mozilla.org/docs/Web/JavaScript/Reference/Global_Objects/Promise)<{publicKey: [string](https://developer.mozilla.org/docs/Web/JavaScript/Reference/Global_Objects/String), address: [string](https://developer.mozilla.org/docs/Web/JavaScript/Reference/Global_Objects/String)}>** an object with a publicKey, address and (optionally) chainCode
+
+
+---
```

Comment:
that block must be placed BEFORE the `# API`, otherwise it gets rewritten

Reply:
OK! No problem, I will redo the PR.